src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -1695,12 +328,29 @@ class DistributedClosestPoint
    * \brief Computes the closest point on the object mesh for each point
    * on the provided query mesh
    *
-   * \param [in] query_node conduit node containing the query points
-   * \param [in] coordset The name of the coordinates within query_node
+   * \param [in] query_node The root node of a mesh blueprint for the query points
+   * Can be empty if there are no query points for the calling rank
+   * \param [in] topology The name of the topology within query_node
+   *
+   * @c queryMesh should have data on the host, regardless of the runtime
+   * policy setting.  Data will be copied to device as needed.
+   *
+   * On completion, the query mesh contains the following fields:
+   *   - cp_rank: Will hold the rank of the object point containing the closest point
+   *   - cp_domain_index: will hold the index of the object domain containing
```

Comment:
```suggestion
   *   - cp_domain_index: Will hold the index of the object domain containing
```

Consistency with other lines.